### PR TITLE
feat: improve Profile State

### PR DESCRIPTION
**Pull request by lucas-bremond**
Symmetrize `Profile::State` with `Trajectory::State`.

### CODE REVIEW

File: bindings/python/test/flight/profile/test_state.py [hunk 1]
```diff
@@ -22,173 +23,146 @@
...
+        # angular_velocity = np.array(angular_velocity)
 
-    # get_velocity
-    ans_velocity = state.get_velocity()
+        # state: State = State(instant, position, velocity, quaternion, angular_velocity, frame)
 
-    assert ans_velocity is not None
-    assert isinstance(ans_velocity, np.ndarray)
-    assert np.array_equal(ans_velocity, velocity)
+        # assert isinstance(state, State)
```
Comment (robinpdm) [inline bindings/python/test/flight/profile/test_state.py:120]: TBR?

File: bindings/python/test/flight/system/dynamics/test_satellite_dynamics.py [hunk 2]
```diff
@@ -11,8 +11,9 @@
 
 import numpy as np
 
-import ostk.physics as physics
 import ostk.mathematics as mathematics
+
+import ostk.physics as physics
 import ostk.astrodynamics as astrodynamics
```
Comment (robinpdm) [inline bindings/python/test/flight/system/dynamics/test_satellite_dynamics.py:17]:
```suggestion
import ostk.physics as physics

import ostk.astrodynamics as astrodynamics
```

File: bindings/python/test/flight/test_profile.py [hunk 3]
```diff
@@ -32,96 +33,79 @@
...
 
-#     assert profile is not None
-#     assert isinstance(profile, Profile)
-#     assert profile.is_defined()
+        assert profile is not None
+        assert isinstance(profile, Profile)
+        assert profile.is_defined() is False
 
-################################################################################################################################################################
+    # def test_inertial_pointing (self):
```
Comment (robinpdm) [inline bindings/python/test/flight/test_profile.py:87]: Unrelated to these changes, but should these be commented out / skipped?

Comment (lucas-bremond) [reply bindings/python/test/flight/test_profile.py]: Fixed!

File: bindings/python/test/flight/system/dynamics/test_satellite_dynamics.py (hunk 2 above)
Comment (lucas-bremond) [reply bindings/python/test/flight/system/dynamics/test_satellite_dynamics.py]: Fixed!

File: bindings/python/test/flight/profile/test_state.py (hunk 1 above)
Comment (lucas-bremond) [reply bindings/python/test/flight/profile/test_state.py]: Fixed!

**Review by robinpdm — APPROVED** [review]

**Pull request merged**